### ISSUE TITLE
Any Updates?

### ISSUE THREAD

**Issue by schneidr** [PapirusDevelopmentTeam/papirus_icons#25]
Is this icon pack going to be updated again? The main icon pack for Linux regularly receives updates, but the Android pack is pretty far behind.

I would imagine that it shouldn't be too difficult to update the Android icon pack automatically when the main pack is updated.

**Comment by SmartFinn** [comment]
I don't think so. It definitely doesn't happen soon. There are nothing common between papirus-icon-theme and this project. I haven't so much time and enthusiasm to support both.

**Comment by Cyanide-1** [comment]
I hope this gets maintained again one daym.

**Comment by irasponsible** [comment]
Even if no android-specific icons get added, packing the existing desktop icons in would be good!

I'm willing to manually assign icons to programs, but there's just so many fewer icons than on desktop.

I tried just copying them all to my phone, but apparently Nova expects pngs, so I'll have to bulk convert them.

**Comment by SteavenGamerYT** [comment]
please update them

**Comment by ahydronous** [comment]
@SmartFinn probably worth it to set this repo to "archived" status then so people know not to expect anything

**Comment by capsudo** [comment]
I made a fork that is updated with icons from [Papirus icon theme repo](https://github.com/PapirusDevelopmentTeam/papirus-icon-theme) (v20260801) => https://github.com/capsudo/papirus-android

apk available in Releases
